packages/waku/src/lib/builder/build.ts

```diff
@@ -237,7 +238,11 @@ const buildServerBundle = async (
         ? [
             rscServePlugin({
               ...config,
-              distServeJs: DIST_SERVE_JS,
+              distServeJs: serve !== 'cloudflare'
+                ? DIST_SERVE_JS
+                // Cloudflare Pages advanced mode requires the dist entry file
+                // to be named `_worker.js`
```

Comment:
Yeah for sure. Astro's CF adapter creates a build output that looks like this:

![image](https://github.com/user-attachments/assets/ee6ce935-cb73-494a-bacf-5a25ee5bc68f)

I don't know how they are using a custom entry path to `_worker/index.js` unless I browse the source code. Because CFP docs say the entry must be called `_worker.js` and be in the output root directory. 

But the point is it means it is possible to customize the `_worker.js` entry file 👍